src/templates/location.tsx

```diff
@@ -5,7 +5,7 @@
  * is generated for every corresponding (based on the filter) stream document.
  *
  * Another way to think about it is that a page will be generated using this
- * template for every eligible entity in your Knowledge Graph.
+ * template for every eligible entity in your Knowledge Graph
```

Comment:
Any reason you removed the periods from these sentences?

Reply:
I didn't do that. Maybe it was the formatter?!

Reply:
Oh I know why!! It was to trigger a rebuild when I was doing the fleet deploys...lol